resources/sts/4.17/hypershift/openshift_hcp_shared_vpc_route_53_credentials_policy.json

```diff
@@ -0,0 +1,45 @@
+{
+  "Version" : "2012-10-17",
+  "Statement" : [
+    {
+      "Sid" : "ReadPermissions",
+      "Effect" : "Allow",
+      "Action" : [
+        "elasticloadbalancing:DescribeLoadBalancers",
+        "route53:GetHostedZone",
+        "route53:ListResourceRecordSets",
```

Comment:
Need route53:ListHostedZones for ingress operator to function properly

Reply:
Updated!